lib/bitstyles_phoenix/showcase.ex

```diff
@@ -54,11 +54,13 @@ defmodule BitstylesPhoenix.Showcase do
           srcdoc:
             ~s(<html style="background-color: transparent;"><head><style>@media (prefers-color-scheme: dark\){body{color: #fff;}}</style><link rel="stylesheet" href="#{
               dist
-            }/build/bitstyles.css"></head><body>#{extra_html |> String.replace("\n", "")}#{result}</body></html>),
+            }/build/bitstyles.css"></head><body>#{
+              Enum.join([extra_html, result]) |> String.replace("\n", "")
+            }</body></html>),
           # https://stackoverflow.com/questions/819416/adjust-width-and-height-of-iframe-to-fit-with-content-in-it
           onload:
             "javascript:(function(o){o.style.height=o.contentWindow.document.body.scrollHeight+\"px\";}(this));",
-          style: "height:1px;width:100%;border:none;overflow:hidden",
+          style: "height:1px;width:100%;border:none;overflow:hidden;margin-left: 1em",
```

Comment:
This makes the rendered examples have some space on the left, so the structure becomes easier on the eyes.